spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -247,6 +247,7 @@
     it 'does nothing if user already exists' do
       Fabricate(:account, username: 'test')
       subject
+      expect(response).to have_http_status(:success)
     end
```

Comment:
Hmm, this test doesn't really test anything except that it doesn't raise an error… I'd rather explicitly test that than the HTTP status that is more confusing than anything.

A longer-term fix would probably be to test that the form is displayed anew with an error (the other tests for this controller do not test that, but instead checked that no account was created, which is not something we can test in this case).

Reply:
While reviewing that feedback, I realized that this spec does not sit within any of the context that actually have a `subject` declared ... so the `subject` of the spec was just the top level class being described, and (I guess?) this leads to a response object with `:success` even when nothing has been called.

I just updated to give it it's own context with a description, and changed the spec to check both the response code and for relevant error text around the invalid username value.